src/SupabaseClient.ts

```diff
@@ -137,7 +137,7 @@ export default class SupabaseClient {
   /**
    * Returns an array of all your subscriptions.
    */
-  getSubscriptions(): RealtimeSubscription[] {
+  _getSubscriptions(): RealtimeSubscription[] {
```

Comment:
@ftonato i would keep it as `getSubscriptions` b/c developers can use it to grab their subscriptions and it'll break existing code (e.g. `client.getSubscriptions()`)

Reply:
@w3b6x9 Sorry for the change! I thought the method was only used internally in the class, hence the idea of initialising it with underscore. Anyway, I "removed" the change and fixed the conflicts...

Reply:
@ftonato no worries and thank you!